content/articles/Introduction-to-Linux-Shell-and-Shell-Scripting/index.md

```diff
@@ -0,0 +1,148 @@
+You interact with shell indirectly if you use any major operating system. At the point when you utilize a terminal on Ubuntu, Linux Mint, or some other Linux dissemination, you are speaking with shell. In this article, I will describe Linux shells and shell scripting; however, before we can comprehend shell scripting, we must first learn the following terms: **Kernel**, **Shell**, **Terminal**.
+
+### Table of contents
+
+- [What is the kernel](#what-is-the-kernel)
+- [What is Shell](#what-is-shell)
+- [Shell Scripting](#shell-scripting)
+- [Why do we need shell scripts](#why-do-we-need-shell-scripts)
+- [Easy demo of Bash Shell scripting](#easy-demo-of-bash-shell-scripting)
+
```

Comment:
Please include a prerequisite section for the reader. It is important especially for beginners